src/main/scala/com/koverse/example/spark/NaiveBayesTrainedTransform.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2016 Koverse, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.koverse.example.spark
+
+import java._
+
+import com.koverse.com.google.common.collect.Lists
+import com.koverse.sdk.Version
+import com.koverse.sdk.data.{Parameter, SimpleRecord}
+import com.koverse.sdk.transform.spark.{JavaSparkTransform, JavaSparkTransformContext}
+import org.apache.spark.mllib.classification.{NaiveBayes, NaiveBayesModel}
+import org.apache.spark.mllib.regression.LabeledPoint
+import com.koverse.sdk.transform.spark.sql.{KoverseSparkSql}
+import org.apache.spark.api.java.JavaRDD
+import org.apache.spark.ml.feature.Tokenizer
+import org.apache.spark.ml.feature.HashingTF
+import org.apache.spark.mllib.linalg.{SparseVector, Vectors}
+import org.apache.spark.sql.DataFrame
+
+
+class NaiveBayesTrainedTransform extends JavaSparkTransform {
+
+  override def execute(context: JavaSparkTransformContext): JavaRDD[SimpleRecord] = {
+
+    val inputCollectionId = context.getInputCollectionIds().get(0)
+    val SQLContext = KoverseSparkSql.createSqlContext(context.getJavaSparkContext.sc)
+    val inputDataFrame:DataFrame = KoverseSparkSql.createDataFrame(context.getInputCollectionRdds().get(inputCollectionId),
+      SQLContext , context.getInputCollectionSchemas().get(inputCollectionId))
+
+    val tokenizer:Tokenizer = new Tokenizer().setInputCol("Weather").setOutputCol("words")
```

Comment:
This block of code looks like it is a duplicate of the getData function in the other Transform. Could we refactor it to a common class/function?